### PR TITLE
Update webchat dependency and adds default values for the new webchat settings

### PR DESCRIPTION
**Pull request by stu-greenshoots**
This PR pulls in the webchat dependency to add the new webchat settings and sets default values for them

### CODE REVIEW

**Review by elliotmassen — APPROVED** [review]
This all works as expected! There's an issue related to the download button which I've mentioned in the [webchat PR](https://github.com/opendialogai/webchat/pull/321#discussion_r662193802).

One question I have is whether we have a way to granularly apply new webchat settings to instances. If we don't persist new settings, we'll fallback on the defaults in webchat until the user updates their webchat settings, at which point the new settings will be persisted. At the moment I think our only option is `php artisan webchat:setup` which would reset other settings that users may have already configured. Perhaps we need to alter that command to have an `-a` flag to set all the settings, and to allow something like `php artisan webchat:setup showEndChatButton showRestartButton` for granular settings.

File: app/Console/Commands/SetWebchatSettings.php [hunk 1]
```diff
@@ -113,6 +113,8 @@ public function handle()
             WebchatSetting::NUMBER_OF_MESSAGES => 10,
             WebchatSetting::COLLECT_USER_IP => true,
             WebchatSetting::SHOW_RESTART_BUTTON => false,
+            WebchatSetting::SHOW_DOWNLOAD_BUTON => false,
```
[the remark this answers is not in the document]
Comment (stu-greenshoots) [reply app/Console/Commands/SetWebchatSettings.php:116]: yeah, good catch, I hadn't really thought about that one

**Review by elliotmassen — APPROVED** [review]

File: app/Console/Commands/SetWebchatSettings.php [hunk 2]
```diff
@@ -22,26 +22,27 @@ class SetWebchatSettings extends Command
...
+        if (!$this->option('non-interactive')) {
+            if (!$settingsInput && !$this->confirm('This will overwrite all current settings - are you sure you want to run?')) {
+                $this->info('OK, not running');
+                return 1;
+            } elseif (!$this->confirm(
+                sprintf('This will update all of these settings - are you sure: %s', implode(',', $settingsInput))
+            )
+            ) {
+                $this->info('OK, not running');
+                return 1;
```
Comment (elliotmassen) [inline app/Console/Commands/SetWebchatSettings.php:42]: If I run the command as `php artisan webchat:setup`, I have two answer "yes" twice for the rest of the command to run - perhaps the conditions here aren't quite right?

**Review by elliotmassen — COMMENTED** [review]
Love the new aspects of the `webchat:setup` command ❤️  Just one issue below.

File: app/Console/Commands/SetWebchatSettings.php (hunk 2 above)
Comment (stu-greenshoots) [reply app/Console/Commands/SetWebchatSettings.php]: Ah yeah, its totally set up to ask you twice 🤦 
Fixed now

**Pull request merged**